src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -26,127 +17,135 @@
-    font: 11px Verdana, Arial, sans-serif;
+    padding: 0 36px 0 0;
+    position: fixed;
+    right: 0;
     text-align: left;
-    color: #2f2f2f;
-
-    background-image: -moz-linear-gradient(top, #e4e4e4, #ffffff);
-    background-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#e4e4e4), to(#ffffff));
-    background-image: -o-linear-gradient(top, #e4e4e4, #ffffff);
-    background: linear-gradient(top, #e4e4e4, #ffffff);
-
-    bottom: 0;
-    border-top: 1px solid #bbb;
+    z-index: 99999;
 }
 .sf-toolbarreset abbr {
-    border-bottom: 1px dotted #000000;
-    cursor: help;
-}
-.sf-toolbarreset span,
-.sf-toolbarreset div,
-.sf-toolbarreset td,
-.sf-toolbarreset th {
-    font-size: 11px;
+    border: dashed #777;
+    border-width: 0 0 1px;
 }
+.sf-toolbarreset svg,
 .sf-toolbarreset img {
-    width: auto;
-    display: inline;
-}
-.sf-toolbarreset table {
-    border-collapse: collapse;
-    border-spacing: 0;
-    background-color: #000;
-    margin: 0;
-    padding: 0;
-    border: 0;
-    width: 100%;
-    table-layout: auto;
+    max-height: 20px;
 }
+
 .sf-toolbarreset .hide-button {
+    background: #444;
+    cursor: pointer;
     display: block;
     position: absolute;
     top: 0;
     right: 0;
-    width: 40px;
-    height: 40px;
+    width: 36px;
+    height: 36px;
     cursor: pointer;
-    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA8AAAAPCAMAAAAMCGV4AAAAllBMVEXDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PDw8PExMTPz8/Q0NDR0dHT09Pb29vc3Nzf39/h4eHi4uLj4+P6+vr7+/v8/Pz9/f3///+Nh2QuAAAAIXRSTlMABgwPGBswMzk8QktRV4SKjZOWmaKlq7TAxszb3urt+fy1vNEpAAAAiklEQVQIHUXBBxKCQBREwRFzDqjoGh+C2YV//8u5Sll2S0E/dof1tKdKM6GyqCto7PjZRJIS/mbSELgXOSd/BzpKIH1ZefVWpDDTHsi8mZVnwImPi5ndCLbaAZk3M58Bay0h9VbeSvMpjDUAHj4jL55AW1rxN5fU2PLjIgVRzNdxVFOlGzvnJi0Fb1XNGBHA9uQOAAAAAElFTkSuQmCC');
-    background-repeat: no-repeat;
-    background-position: 13px 11px;
+    text-align: center;
+}
+.sf-toolbarreset .hide-button svg {
+    max-height: 18px;
+    padding-top: 10px;
 }
 
 .sf-toolbar-block {
-    white-space: nowrap;
-    color: #2f2f2f;
+    cursor: default;
     display: block;
-    min-height: 28px;
-    border-bottom: 1px solid #e4e4e4;
-    border-right: 1px solid #e4e4e4;
-    padding: 0;
     float: left;
-    cursor: default;
+    height: 36px;
+    margin-right: 0;
+    white-space: nowrap;
+}
+.sf-toolbar-block > a,
+.sf-toolbar-block > a:hover {
+    display: block;
+    text-decoration: none;
 }
 
 .sf-toolbar-block span {
     display: inline-block;
 }
+.sf-toolbar-block .sf-toolbar-value {
+    color: #F5F5F5;
+    font-size: 13px;
+    line-height: 36px;
+    padding: 0;
+}
+.sf-toolbar-block .sf-toolbar-label,
+.sf-toolbar-block .sf-toolbar-class-separator {
+    color: #AAA;
+    font-size: 12px;
+}
 
+.sf-toolbar-block .sf-toolbar-info {
+    border-collapse: collapse;
+    display: table;
+    z-index: 99999;
```

Comment:
the toolbar info should have a higher zindex than the toolbar itself to place it properly when the toolbar spans on 2 lines (ensuring that the info does not get partially hidden by the first row of the toolbar)